accounts/abi/bind/bind.go

```diff
@@ -594,6 +601,24 @@ func decapitalise(input string) string {
 	return strings.ToLower(goForm[:1]) + goForm[1:]
 }
 
+// convertToNil converts any type to its proper nil form.
+func convertToNil(input abi.Type) string {
+	switch input.T {
+	case abi.IntTy, abi.UintTy:
+		return "0"
+	case abi.StringTy:
+		return "\"\""
+	case abi.BoolTy:
+		return "false"
+	case abi.AddressTy:
+		return common.Address{}.String()
+	case abi.HashTy:
+		return common.Hash{}.String()
+	default:
```

Comment:
Ah my mistake on the last comment but since the empty value needs to be returned and output in the code, it looks like these need to be `"common.Address{}"` and `"common.Hash{}"` so that it shows up as those strings in the code.

Reply:
Another alternative here would be to change the precompile template to always use `UnpackIntoInterface` so that we can avoid this logic entirely of needing to know what version of the empty value to use.

For now, I think we can make the update here to use `"common.Address{}"` and `"common.Hash{}"` and then make separate tickets for the remaining items from here: https://github.com/ava-labs/subnet-evm/issues/227 since it will already be a huge improvement to get this in.